find_courses.py

```diff
@@ -1,15 +1,15 @@
 
 import psycopg2
 import sys, os
-sys.path.insert(0, './src/')
+sys.path.insert(0, SRC)
 import json
 import webbrowser as wb
 import methods
-
+import consts
 def main():
     
     #consts
```

Comment:
this whole consts section is unnecessary. please most padding and limit into consts file, and then use CONN_STRING everywhere. conn_string = conn_string is redundant